gulpfile.js

```diff
@@ -14,7 +14,7 @@ gulp.task('nodemon', () => {
 gulp.task('server', ['nodemon'], () => {
     browserSync.init({
         proxy: "http://localhost:3000",
-        port: 7000,
+        port: 7001,
```

Comment:
hello @andela-ooduye thats a very good question. local host 7000 is not functional on my laptop for some weird reasons  so i had to change it on other to be able to test the app on my system